src/main/java/bgu/spl/net/impl/stomp/StompEncDec.java

```diff
@@ -0,0 +1,82 @@
+package bgu.spl.net.impl.stomp;
+
+import bgu.spl.net.api.MessageEncoderDecoder;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public class StompEncDec implements MessageEncoderDecoder<Frame> {
+
+    private byte[] bytes = new byte[1 << 10]; //start with 1k
+    private int len = 0;
+
+    /**
+     *   Converts a String to the Frame object representing it
+     * @param msg The message as a string
+     * @return The Frame representation
+     */
+    private Frame parseMessage(String msg) {
+        int delim = 0, prevDelim = 0;
+        int msgPart = 1;
+        Frame parsed = null;
+        do {
+            prevDelim = delim;
+            delim = msg.indexOf("\n", prevDelim+1); // Find the end of the next line
+            if (msgPart == 1) { // The command
```

Comment:
save it as a const, don't use magic numbers 😄

Reply:
Done

Reply:
meant the msgPart == 1 :sweat_smile: